build/Dockerfile.ci

```diff
@@ -3,28 +3,26 @@
 # building the operator from the PR source without using the operator-sdk.
 # NOTE: RUN mkdir <dir> is being used in cases where a directory needs to be created as CI image builds does not seem to
 # be creating the directory as a result of the WORKDIR directive. 
-# build stage for building binaries
 
-# We cannot use a golang image as building the kubelet requires rsync and that is not present plus there is no easy way
-# to install it.
-FROM registry.access.redhat.com/ubi8/ubi-minimal as build
+# build stage for building binaries
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 as build
```

Comment:
@LorbusChris please check with ART if they will be updating this image too. If they are not please open a ticket requesting them to do so.